src/librustdoc/html/static/rustdoc.css

```diff
@@ -1763,7 +1770,7 @@ details.rustdoc-toggle[open] > summary.hideme > span {
 }
 
 details.rustdoc-toggle[open] > summary::before {
-	content: "[−]";
+	content: "[-]";
```

Comment:
Why the change here? I think I modeled the longer dash over what was being done with the JS-style toggles previously. I don't particularly object but it seems like an unrelated diff.

Reply:
Oh it's because the rendering was broken in puppeteer. Do you want me to revert it?

Reply:
Honestly I don't have an opinion - I just made sure to use the existing long dash because I assumed you cared about it. :-) Fine to leave as-is, but should have an explanation in a commit message and/or the PR description.

Reply:
I'm pretty sure the idea is that + and − have exactly the same width. We don't want the toggle to shift around when you click it.

Reply:
Good point, gonna revert this change then.

Reply:
Compare these three ways of writing...

# 3+3| plus (U+002D)
# 3–3| minus (U+2212)
# 3-3| hyphen-minus (U+002B)

On my machine, at least, the first two are perfectly aligned, and the last is not. This is probably the same in most fonts.